Dockerfile

```diff
@@ -1,9 +1,9 @@
 ########################################################################
-# Dockerfile for Oracle JDK 8 on Ubuntu 16.04
+# Dockerfile for JDK on Ubuntu 22.04 LTS
 ########################################################################
 
 # pull base image
-FROM ubuntu:16.04
+FROM ubuntu:22.04
```

Comment:
Shouldn't we use the latest LTS, i.e., 24.04?

https://ubuntu.com/about/release-cycle